src/react-apps/applications/shared/__tests__/LeftDrawerMenu.test.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+import { MemoryRouter } from 'react-router';
+import * as renderer from 'react-test-renderer';
+import LeftDrawerMenu from '../src/navigation/drawer/LeftDrawerMenu';
+
+describe('render left drawer menu', () => {
+  let mockMenuType: string;
+  let mockactiveLeftMenuSelection: string;
+
+  beforeEach(() => {
```

Comment:
Only one beforeEach which covers all tests inside 'describe'.
If you want to adjust the variables, do so inside the 'it' function.

Reply:
fixed